packages/predicate/src/test-predicate-basic/Forc.lock

```diff
@@ -0,0 +1,12 @@
+[[package]]
+name = 'core'
+dependencies = []
+
+[[package]]
+name = 'std'
+source = 'git+https://github.com/fuellabs/sway?tag=v0.14.5#60c626cf486d5c53c44d84db1ec81cb90174cad3'
```

Comment:
Not a fan of how this uses a hard-coded and old version of `forc`. But this is a repo-wide issue I think?

Reply:
It is what it is